Lib/logging/__init__.py

```diff
@@ -438,7 +442,7 @@ def validate(self):
             self._invalid_raise()
 
     def _invalid_raise(self, additional_message=None):
-        msg = 'Invalid format %s for %s style' % (self._fmt, self.default_format[0])
+        msg = "Invalid format '%s' for '%s' style" % (self._fmt, self.default_format[0])
```

Comment:
You can just use `'Invalid format %r for %r style'` to get the single quotes in.